client/lxd_network_addresses.go

```diff
@@ -0,0 +1,24 @@
+package lxd
+
+import (
+	"fmt"
+
+	"github.com/lxc/lxd/shared/api"
+)
+
+// GetNetworkACLs returns a list of Network ACL structs.
+func (r *ProtocolLXD) GetNetworkAddresses() ([]api.NetworkAddresses, error) {
+	if !r.HasExtension("network_addresses") {
```

Comment:
Please use r.CheckExtension which avoids the duplication of the API extension string.

Reply:
sure